packages/hydrogen/package.json

```diff
@@ -38,6 +38,7 @@
         "default": "./dist/production/index.js"
       }
     },
+    "./server": "./dist/production/server.js",
```

Comment:
@frandiox Not sure this is the ideal way to create a server export. We could also separate this into another PR, but thought I would do it here to bring it to our attention.